it/uk/gov/hmrc/minorentityidentificationfrontend/views/uaViews/UaCheckYourAnswersSpecificViewTests.scala

```diff
@@ -43,10 +43,10 @@ trait UaCheckYourAnswersSpecificViewTests {
       "have a utr row" in {
         val utrRow = summaryListRows.head
 
-        utrRow.getSummaryListQuestion mustBe messages.utr
+        utrRow.getSummaryListQuestion mustBe messages.uaNoUtr
```

Comment:
uaNoUtr sounds like there is no utr present. Maybe rename to be slightly clearer e.g. uaUtr

Reply:
sorry, I did it when message key was changing. I change it now thanks

Reply:
done